src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -64,16 +65,34 @@ public void updateUser(final DBUser user, final HashedPassword hashedPassword) {
 
     try (final Connection con = connectionSupplier.get();
         final PreparedStatement ps =
-            con.prepareStatement("update ta_users set password = ?,  email = ?, admin = ? where username = ?")) {
+            con.prepareStatement("update ta_users set password = ?,  email = ? where username = ?")) {
       ps.setString(1, hashedPassword.value);
       ps.setString(2, user.getEmail());
-      ps.setInt(3, user.isAdmin() ? 1 : 0);
-      ps.setString(4, user.getName());
+      ps.setString(3, user.getName());
       ps.execute();
       con.commit();
     } catch (final SQLException e) {
-      throw new IllegalStateException(
-          "Error updating name:" + user.getName() + " email: " + user.getEmail() + " pwd: " + hashedPassword.mask(), e);
+      throw new IllegalStateException(String.format("Error updating name: %s email: %s pwd: %s",
+          user.getName(), user.getEmail(), hashedPassword.mask()), e);
+    }
+  }
+
+  /**
+   * A method similar to update user, used by tests only.
+   * Does only affect the admin state of the given user.
+   * The DB is updated with user.isAdmin()
+   */
+  public void makeAdmin(final DBUser user) {
```

Comment:
Since this method is only used for testing, can you please add a `@VisibleForTesting` annotation here?